### PR TITLE
*: cleanup pd interface, remove cluster id.

### PR DESCRIPTION
**Pull request by siddontang**
@ngaut @BusyJay @disksing @tiancaiamao

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM.

File: tests/raftstore/test_pd.rs [hunk 1]
```diff
@@ -19,21 +19,21 @@ use super::util;
 #[test]
 fn test_pd_heartbeat() {
     let mut cluster = new_server_cluster(0, 1);
-    cluster.cfg.store_cfg.pd_heartbeat_tick_interval = 50;
+    cluster.cfg.store_cfg.pd_heartbeat_tick_interval = 10;
```
Comment (disksing) [inline tests/raftstore/test_pd.rs]: Is this change intentional or by mistake?

Comment (siddontang) [inline tests/raftstore/test_pd.rs]: I change it before but forget to send to another PR.

**Comment by disksing** [comment]
LGTM.
